src/EditorFeatures/CSharpTest/Completion/CompletionProviders/ObjectCreationCompletionProviderTests.cs

```diff
@@ -438,6 +439,58 @@ void M1()
             await VerifyItemExistsAsync(markup, "object");
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        [WorkItem(4585, "https://github.com/dotnet/roslyn/issues/4585")]
+        public async Task GenericArguments_PredefinedType()
+        {
+            var markup = @"
+using System;
+
+class A<T> { }
+
+class B
+{
+    A<Int32> _a = new $$
+}";
+
+            await VerifyItemExistsAsync(markup, "A<int>",
+                options: Option(CodeStyleOptions.PreferIntrinsicPredefinedTypeKeywordInDeclaration, LanguageNames.CSharp, true));
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        [WorkItem(4585, "https://github.com/dotnet/roslyn/issues/4585")]
+        public async Task GenericArguments_FrameworkType()
+        {
+            var markup = @"
+using System;
+
+class A<T> { }
+
+class B
+{
+    A<int> _a = new $$
+}";
+
+            await VerifyItemExistsAsync(markup, "A<Int32>",
+                options: Option(CodeStyleOptions.PreferIntrinsicPredefinedTypeKeywordInDeclaration, LanguageNames.CSharp, false));
+        }
+
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/13610"), Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task EscapeAwaitIdentifierInAsyncMethod()
```

Comment:
Skip?
In the PR you said this was fixed:
> Fixes a handful of issues around escaping identifiers
In C#, await identifiers weren't being escaped in async methods

Reply:
Some cases were fixed, some remain broken. Those that insert "minimal" text are still broken because of #13610, while those that insert non-minimal strings are fixed by these changes.